AirCasting/Dashboard/DashboardView.swift

```diff
@@ -74,6 +74,12 @@ struct DashboardView: View {
             try! coreDataHook.setup(selectedSection: self.selectedSection.selectedSection)
         }
         .onAppear {
+            try! coreDataHook.setup(selectedSection: .mobileActive)
```

Comment:
why do this when you override in line `83` ?

Reply:
I still don't get it. Can you try with removing this line and check if it works as expected still?